inference-engine/tests/functional/plugin/gna/pass_tests/padded2valid_conv.cpp

```diff
@@ -0,0 +1,307 @@
+            last_op = std::make_shared<Transpose>(maxpool, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvBcastAddActTransp:
+        {
+            auto bcast_add = std::make_shared<Add>(conv, bias_const);
+            auto activation = std::make_shared<Relu>(bcast_add);
+            last_op = std::make_shared<Transpose>(activation, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvBcastAddMaxPoolActTransp:
+        {
+            auto bcast_add = std::make_shared<Add>(conv, bias_const);
+            auto max_pool = std::make_shared<MaxPool>(bcast_add, Strides{ 1, 1 }, Shape{ 0, 0 }, Shape{ 0, 0 }, maxpool_shape);
+            auto activation = std::make_shared<Relu>(max_pool);
+            last_op = std::make_shared<Transpose>(activation, transpose_out_order);
+        }
+        break;
+
+        case modelType::TranspConvTranspBcastAdd:
+        {
+            bias_const = std::make_shared<Constant>(ngPrc, transp_bias_shape);
+            last_op = std::make_shared<Add>(last_op, bias_const);
+        }
+        break;
+
+        case modelType::TranspConvTranspBcastAddAct:
+        {
+            bias_const = builder::makeConstant(ngPrc, transp_bias_shape, bias_weights, true);
+            auto bcast_add = std::make_shared<Add>(last_op, bias_const);
+            last_op = std::make_shared<Relu>(bcast_add);
+        }
+        break;
+
+        case modelType::TranspConvTransp:
+        default:
+            break;
+        }
+
+        function = std::make_shared<Function>(NodeVector{ last_op }, ParameterVector{ input });
+    }
+};
+
+class GnaPadded2ValidConvTest : public Padded2ValidConvTest, GnaLayerTestCheck {
+protected:
+    void Run() override {
+        GnaLayerTestCheck::SkipTestCheck();
+
+        if (!GnaLayerTestCheck::skipTest) {
+            Padded2ValidConvTest::Run();
+        }
+    }
+
+    void SetUp() override {
+        Padded2ValidConvTest::SetUp();
+    }
+};
+
+TEST_P(Padded2ValidConvTest, CompareWithRefs) {
+    Run();
+}
+
+TEST_P(GnaPadded2ValidConvTest, CompareWithRefs) {
+    Run();
+}
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    //InferenceEngine::Precision::FP16
+};
+
+const std::vector<std::map<std::string, std::string>> configs = {
+    {
+        {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
+        {"GNA_SCALE_FACTOR_0", "1"}
+    }
+};
+
+const std::vector<op::PadType> padTypes = {
+        op::PadType::EXPLICIT,
+        op::PadType::SAME_LOWER,
+        //TODO: SAME_UPPER fails for 1d conv
+        //op::PadType::SAME_UPPER,
+        op::PadType::VALID
+};
+
+const std::vector<modelType> models = {
+    modelType::TranspConvTransp,
+    modelType::TranspConvBcastAddTransp,
+    //TODO: this model fails for 1d conv
+    //modelType::TranspConvBcastAddMaxPoolTransp,
```

Comment:
If it fails with error from GNA lib, something like NumOperations == 0, it must be fixed on the master.

Reply:
These tests fail with accuracy, it'll be investigated further in next steps. For now I'd like to leave them here as disabled.